README.md

```diff
@@ -2,18 +2,20 @@
 
 The Reporting Service allows to create HTTP API endpoints using SQL queries and EJS templates and output the result in different formats.
 
-- Create API endpoint description file in `templates/ENDPOINT_NAME.yaml`
-- Create render template in `templates/ENDPOINT_NAME.ejs`
-- See examples in `templates` directory
+- Create Custom Resource for the report.
+  
+  The schema can be found in `helm/reporting-service/crds/mojaloopreport-crd.yaml`
```

Comment:
:tada: schema! Great! Seriously, I hate it when I try to validate some k8s CRD manifest and I can't because the schema isn't published anywhere. I wonder if this could be more visible. Perhaps putting it in the chart NOTES or something might be useful..? Maybe..?

Reply:
And/or in the README.